CMakeLists.txt

```diff
@@ -418,7 +418,7 @@ if(${VELOX_ENABLE_DUCKDB})
 endif()
 
 set_source(fmt)
-resolve_dependency(fmt)
```

Comment:
Not sure this is 100% required, sine we will work with fmt 9.0 too - we are defaulting to fmt 10, but so far we have tried to maintain compatibility with 9+.

Reply:
I would change it to 9 since thats the new baseline.

Reply:
Sure, will do and set it to 9.0.0.
Hopefully, this will prevent users from seeing the build issues and not knowing why.

Reply:
Yes this is good. As mentioned by Deepak in the other PR we should enforce minimum versions for the other deps this way as well. CMake gives a much better error than some confusing compilation issue. (sorry that this wasn't in place until now!)